MAUI/Busy-Indicator/AnimationType.md

```diff
@@ -161,3 +161,33 @@ SfBusyIndicator busyIndicator = new SfBusyIndicator()
 The following gif image illustrates the result of the above code.
 
 ![Double Circle](Images/AnimationType/DoubleCircle.png)
+
+### Globe
+
+The Globe animation is an one of the built-in animations in [SfBusyIndicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1). Refer to the following code example. Here, we’ll set the animation type as `Globe`
```

Comment:
The Globe animation is one of the built-in animations in [.NET MAUI Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1). Refer to the following code example. Here, we’ll set the `AnimationType` as `Globe`

Reply:
Updated and added API link for AnimationType